library/globals.inc.php

```diff
@@ -2789,9 +2800,9 @@ function gblTimeZones()
         'fhir_enable' => array(
             xl('Enable FHIR Provider Client Service'),
             array(
-                0 => xl('Off: No Service.'),
-                1 => xl('On: HAPI FHIR.'),
-                2 => xl('On: Smart on FHIR.'),
+                0 => xl('Off No Service.'),
```

Comment:
As above, the `Off No Service.` label seems odd. Maybe a dash or just remove either Off or the No Service part.